docs/changelog.rst

```diff
@@ -7,6 +7,10 @@ Unreleased
 * Fix restore of database from S3 storage by reintroducing inputfile.seek(0) to utils.uncompress_file
 * Fix bug where dbbackup management command would not respect settings.py:DBBACKUP_DATABASES
 
+4.2.0 (2022-01-30)
+------------------
+* Add PostgreSQL Schema support by @angryfoxx in https://github.com/jazzband/django-dbbackup/pull/507
```

Comment:
Move this to the `unreleased` section above

Reply:
fixed